<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DiffPass V2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 226, 236);
font-family:Segoe-UI;
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="MainWindow_2">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>240</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 42px;
color: #29282f;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>DiffPass v2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>200</y>
        <width>420</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 30px;
color: #29282f;</string>
      </property>
      <property name="text">
       <string>Sing in or create a new account</string>
      </property>
     </widget>
     <widget class="QPushButton" name="login">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>310</y>
        <width>240</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#login{
background-color:#29282f;
border-radius:10px;
font-size: 18px;
color:#fff;
}

QPushButton#login:pressed{
padding-left:5px;
padding-top:5px;
background-color:#595968;
background-position:calc(100%-10px)center;
}

QPushButton#login:hover{
background-color:#595968;
}
</string>
      </property>
      <property name="text">
       <string>Log In</string>
      </property>
     </widget>
     <widget class="QPushButton" name="signup">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>400</y>
        <width>240</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#signup{
background-color:#29282f;
color:#fff;
border-radius:10px;
font-size: 18px;
}

QPushButton#signup:pressed{
padding-left:5px;
padding-top:5px;
background-color:#595968;
background-position:calc(100%-10px)center;
}

QPushButton#signup:hover{
background-color:#595968;
}</string>
      </property>
      <property name="text">
       <string>Sign Up</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ManagerWindow">
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>250</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#29282f;</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 30px;
color:#fff;</string>
       </property>
       <property name="text">
        <string>DiffPass v2</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>250</width>
         <height>3</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>80</y>
         <width>161</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 14px;
color:#fff;</string>
       </property>
       <property name="text">
        <string>PASSWORD MANAGER</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>520</y>
         <width>250</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(65, 64, 75);</string>
       </property>
       <widget class="QPushButton" name="settings">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>31</y>
          <width>32</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#settings{
	background:none;
	border-image: url(:/icons/icons/settings.png);
}

#settings:pressed{
	border-image: url(:/icons/icons/settings_pushed.png);
	background:none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="passwords">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>120</y>
         <width>250</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#passwords{
font-size: 14px;
color:#fff;
}
</string>
       </property>
       <property name="text">
        <string>Passwords</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="generator">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>480</y>
         <width>250</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#generator{
font-size: 14px;
color:#fff;
}
</string>
       </property>
       <property name="text">
        <string>Password Generator</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>650</width>
        <height>61</height>
       </rect>
      </property>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>650</width>
         <height>3</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="remove">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>200</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#remove{
background-color:#29282f;
border-radius:10px;
font-size: 18px;
color:#fff;
}

QPushButton#remove:pressed{
padding-left:5px;
padding-top:5px;
background-color:#595968;
background-position:calc(100%-10px)center;
}

QPushButton#remove:hover{
background-color:#595968;
}
</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
      <widget class="QPushButton" name="add">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>10</y>
         <width>200</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#add{
background-color:#29282f;
border-radius:10px;
font-size: 18px;
color:#fff;
}

QPushButton#add:pressed{
padding-left:5px;
padding-top:5px;
background-color:#595968;
background-position:calc(100%-10px)center;
}

QPushButton#add:hover{
background-color:#595968;
}
</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>61</y>
        <width>270</width>
        <height>539</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(230, 226, 236);

</string>
      </property>
      <widget class="QLabel" name="recordName">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 20px;
font-weight:bold;
color:#29282f;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>250</width>
         <height>180</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:10px;
background-color: rgb(65, 64, 75);
color:#fff;</string>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>40</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#fff;</string>
        </property>
        <property name="text">
         <string>Url</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>61</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>E-mail</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>61</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="recordPass">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>120</y>
          <width>151</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:15px;
background-color:#36383c;
color:#e3e3e3;
border-radius:5px;
padding: 0 10px;


</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="recordUrl">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>40</y>
          <width>151</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:15px;
background-color:#36383c;
color:#e3e3e3;
border-radius:5px;
padding: 0 10px;


</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="recordMail">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>151</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:15px;
background-color:#36383c;
color:#e3e3e3;
border-radius:5px;
padding: 0 10px;


</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>61</y>
        <width>380</width>
        <height>539</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
